app/javascript/components/public_health/PublicHealthHeader.js

```diff
@@ -127,7 +127,8 @@ class PublicHealthHeader extends React.Component {
         <Modal.Body>
           {this.state.importType === 'saf' && (
             <div className="mb-3">
-              <a href="https://github.com/SaraAlert/SaraAlert/blob/master/public/Sara%20Alert%20Import%20Format.xlsx?raw=true">Download formatting guidance</a>
+              <a href="https://github.com/SaraAlert/SaraAlert/blob/master/public/Sara%20Alert%20Import%20Format.xlsx?raw=true">Download formatting guidance</a>{' '}
+              (Updated 12/2/2020)
```

Comment:
Could you also update this date to the actual date this will be release? That will be 12/15 for 1.18.0.

Reply:
Rather than making this a date, why not make it a version? That way we can just use the version variable that is available, and never have to worry about keeping this up to date.

Reply:
I believe when we first discussed with the PO team, we wanted an explicit date so they can see if it changed since they have last updated their scripts. With a version number, they could still see that it changed (if they remember what the version was before), but maybe a bit harder to recognize. Definitely does make this easier, I can propose that to team